index.html

```diff
@@ -16,10 +16,29 @@
 
 <body>
```

Comment:
You forgot to add a viewport meta tag in your index.html and because of that, your media queries won't work